src/App/ImportUshahidiV2/Repositories/ImportMappingRepository.php

```diff
@@ -26,7 +26,14 @@ class ImportMappingRepository /*extends EloquentRepository*/ implements ImportMa
 
     public function create(ImportMapping $model) : int
```

Comment:
From my profiling, it looks a bit like adding the cache-on-write actually made this slower not faster, but I need to find a larger dataset to test on

Reply:
That's interesting we should definitely check to see why that might be